app/templates/components/notification-dropdown.hbs

```diff
@@ -10,20 +10,22 @@
       {{/link-to}}
     </div>
     <div class="ui basic right aligned segment weight-400">
-      <a href="#">{{t 'Mark all as Read'}}</a>
+      <a href="#" {{action 'markAllRead'}}>{{t 'Mark all as Read'}}</a>
     </div>
   </div>
   <div class="ui fluid link celled selection list">
     {{#each notifications as |notification|}}
```

Comment:
@niranjan94 But we are only showing the unread notifications in the dropdown right? This was the behavior before the api when we had mock response. Do we want to change the approach and show all the notifications now? like facebook does? The thing is for now we are filtering the unread notifications in controller and showing it. And clicking on a notification would mark it as read and remove from the list in dropdown.